### PR TITLE
#163426503 - Track comment edit history

### PR DESCRIPTION
**Pull request by omenkish**
#### What does this PR do?
- It enables a user to track his comment edit history
#### How should this be manually tested?
Clone this repo and run npm test

#### What are the relevant pivotal tracker stories?
[#163426503](https://www.pivotaltracker.com/story/show/163426503)
#### Screenshots (if appropriate)
- On desktop
Screen Shot 2019-04-04 at 6 41 37 PM

- on smaller screens
Screen Shot 2019-04-04 at 6 41 17 PM
Screen Shot 2019-04-04 at 6 40 58 PM
Screen Shot 2019-04-04 at 6 40 19 PM
Screen Shot 2019-04-04 at 6 39 52 PM

### CODE REVIEW

File: __test__/actions/commentAction.test.js [hunk 1]
```diff
@@ -138,13 +141,13 @@ describe('Auth action creators test', () => {
...
 
-  it('should create the ADD_BOOKMARK action', async () => {
+  it('should create the ADD_COMMENT action ', async () => {
     sendHttpRequest.mockRejectedValue(getError(401));
     await store.dispatch(getArticleComments(id));
     expect(toast.error).toHaveBeenCalledWith('hello');
   });
 
-  it('should create the ADD_BOOKMARK action', async () => {
+  it('should create the ADD_COMMENT action', async () => {
```
Comment (p-funky) [inline __test__/actions/commentAction.test.js:150]: Thanks for the attention to detail.

File: package.json [hunk 2]
```diff
@@ -53,7 +53,6 @@
     "react-hot-loader": "^4.8.2",
     "react-inlinesvg": "^0.8.4",
     "react-items-carousel": "^2.3.0",
-    "react-modal": "^3.8.1",
```
Comment (p-funky) [inline package.json:52]: Thanks. We need to remove more of stuff we don't use.

File: src/components/comment/CommentItem.jsx [hunk 3]
```diff
@@ -139,7 +148,15 @@ export class CommentItem extends Component {
               <span className="comment-time">
                 {' '}
                 &nbsp; {moment(comment.createdAt).fromNow()}
-              </span>
+              </span>&nbsp;
+              {
+                (comment.updatedAt > comment.createdAt)
+                && <>
```
Comment (p-funky) [inline src/components/comment/CommentItem.jsx]: What's this about please?

File: src/components/containers/SingleArticle.jsx [hunk 4]
```diff
@@ -107,6 +107,7 @@ class SingleArticle extends Component {
       || !this.props.singleArticle.article.articleBody
     ) return delayDisplay;
 
+
     return (
       <>
```
Comment (p-funky) [inline src/components/containers/SingleArticle.jsx:112]: What's this really? 🤔

Comment (omenkish) [reply src/components/containers/SingleArticle.jsx:112]: Fragment. To hold them all in one container instead of having too many nested divs.

Comment (p-funky) [reply src/components/containers/SingleArticle.jsx:112]: Got it.

**Review by bobbylemm — APPROVED** [review]

**Review by primuse — APPROVED** [review]

**Review by EfeAgare — APPROVED** [review]

**Pull request merged**